test/test-network/systemd-networkd-tests.py

```diff
@@ -173,14 +173,17 @@ def test_bond(self):
 
         self.assertTrue(self.link_exits('bond99'))
 
-        self.assertEqual('802.3ad 4',  self.read_link_attr('bond99', 'bonding', 'mode'))
-        self.assertEqual('layer3+4 1', self.read_link_attr('bond99', 'bonding', 'xmit_hash_policy'))
-        self.assertEqual('1000',       self.read_link_attr('bond99', 'bonding', 'miimon'))
-        self.assertEqual('fast 1',     self.read_link_attr('bond99', 'bonding', 'lacp_rate'))
-        self.assertEqual('2000',       self.read_link_attr('bond99', 'bonding', 'updelay'))
-        self.assertEqual('2000',       self.read_link_attr('bond99', 'bonding', 'downdelay'))
-        self.assertEqual('4',          self.read_link_attr('bond99', 'bonding', 'resend_igmp'))
-        self.assertEqual('1',          self.read_link_attr('bond99', 'bonding', 'min_links'))
+        self.assertEqual('802.3ad 4',         self.read_link_attr('bond99', 'bonding', 'mode'))
+        self.assertEqual('layer3+4 1',        self.read_link_attr('bond99', 'bonding', 'xmit_hash_policy'))
+        self.assertEqual('1000',              self.read_link_attr('bond99', 'bonding', 'miimon'))
+        self.assertEqual('fast 1',            self.read_link_attr('bond99', 'bonding', 'lacp_rate'))
+        self.assertEqual('2000',              self.read_link_attr('bond99', 'bonding', 'updelay'))
+        self.assertEqual('2000',              self.read_link_attr('bond99', 'bonding', 'downdelay'))
+        self.assertEqual('4',                 self.read_link_attr('bond99', 'bonding', 'resend_igmp'))
+        self.assertEqual('1',                 self.read_link_attr('bond99', 'bonding', 'min_links'))
+        self.assertEqual('1218',              self.read_link_attr('bond99', 'bonding', 'ad_actor_sys_prio'))
```

Comment:
This one fails on CentOS:
```sh
======================================================================
FAIL: test_bond (__main__.NetworkdNetDevTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./systemd-networkd-tests.py", line 184, in test_bond
    self.assertEqual('1218',              self.read_link_attr('bond99', 'bonding', 'ad_actor_sys_prio'))
AssertionError: '1218' != '65535'
- 1218
+ 65535
```
I'm wondering if it is possible to make the test aware of the features supported by the kernel so that unsupported bits could be skipped.

Reply:
It seems to be failing with the same error on Fedora with ` 4.18.14-200.fc28.x86_64` too.

Reply:
Interesting, I will run this again. Must have done something wrong here.